### PR TITLE
Added ctrl+enter and shift+enter in the url text box.

### PR DESCRIPTION
**Pull request by rohitsakala**
Basically, Ctrl + enter key shortcut adds 'www.' prefix and '.com' as suffix to the entered text in the url text box.
Shift+enter key shortcut adds 'www.' as prefix and '.net' to the entered text in the url text box.

### CODE REVIEW

File: webtoolbar.py [hunk 1]
```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
         selection = self._search_view.get_selection()
         model, selected = selection.get_selected()
 
+        if event.get_state() & Gdk.ModifierType.CONTROL_MASK:
+            if keyname == 'Return':
+                m = re.search('\.', self.props.text)
```
Comment (SAMdroid-apps) [inline webtoolbar.py]: You can use `m = '.' in self.props.text` and then you don't waste compute power with regex.

File: webtoolbar.py [hunk 2]
```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
...
+                if self.props.text == "":
```
Comment (SAMdroid-apps) [inline webtoolbar.py]: or just `if self.props.text:`

File: webtoolbar.py [hunk 3]
```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
...
+                    return False
+                if m is None:
+                    self._set_text("www."+self.props.text+".com")
```
Comment (SAMdroid-apps) [inline webtoolbar.py]: 1.  I don't think we need to add `www.`.  Websites work fine without it and we don't want to ruin any subdomains the user has entered.

2.  Lets take some ideas from the iOS keyboard.  If the user has typed `google` we should add `.com` (as you already do).  But if the user has typed `google.` we should only add `com`.

File: webtoolbar.py [hunk 4]
```diff
@@ -209,6 +209,28 @@ def __key_press_event_cb(self, entry, event):
...
+                    self.activate(self.props.text)
+                    return True
+                self.activate(self.props.text)
+
+        if event.get_state() & Gdk.ModifierType.SHIFT_MASK:
```
Comment (SAMdroid-apps) [inline webtoolbar.py]: All the stuff I said about the `.com` is the same with `.net`

**Comment by SAMdroid-apps** [comment]
Do any other browsers do this?

The `.com` shortcut is pretty logically.  But why `.net` - why isn't it `.org`?

**Comment by godiard** [comment]
I am not convinced that this is a feature we really need in Browse.

**Pull request closed without merging**